xrpl/transaction/types/issued_currency.go

```diff
@@ -0,0 +1,13 @@
+package types
+
+type IssuedCurrency struct {
+	Currency string `json:"currency"`
+	Issuer   string `json:"issuer"`
```

Comment:
Should it be `json:"issuer,omitempty"` as it can be empty if the currency is XRP ?
Also should it be of `types.Address` ?

Reply:
the issuer must not be empty, check https://github.com/XRPLF/xrpl.js/blob/9211ff6e48fd919c2a90716e856c1a5fe305cfba/packages/xrpl/src/models/common/index.ts#L12. Changing to `types.Address`